README.adoc

```diff
@@ -25,38 +26,75 @@ endif::[]
 
 = UnifiedPush Operator
 
-The UnifiedPush Operator for Kubernetes provides an easy way to
-install and manage an AeroGear UnifiedPush Server on Kubernetes.
-
 toc::[]
 
-== Limitations
+== Overview
 
-// https://issues.jboss.org/browse/AEROGEAR-9162
-[NOTE]
-====
-This operator currently only works on OpenShift. This is because it
-provisions `Route`s and `ImageStream`s, and also relies on the fact that OpenShift
-provides an OAuth server. In future we aim to make it work on vanilla
-Kubernetes also.
-====
+The UnifiedPush Operator for Kubernetes provides an easy way to install and manage an https://aerogear.org/docs/unifiedpush/[AeroGear UnifiedPush] Server on OpenShift.
+
+== Prerequisites
```

Comment:
The previous structure of this readme had the installation section here for "users" who wanted to install the operator, and then there was a section later called "Development", for people who wanted to make changes to the project. Which one of those are these "Prerequisites" for? To use the operator the only thing you need from this list is kubectl (and you could even get by without that :wink: )

Reply:
To run the make commands which make the first contact really more friendly, I think so. 
The changed performed as based on the difficulties that I was a dev as well had to understand it. 

Any project tries to make easier the first contact, which in my point of view is by describing to use the make commands and installing the default configuration of the project. Then after check, it running we can try to customize it.